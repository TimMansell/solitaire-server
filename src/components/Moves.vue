<template>
  <div>Moves: {{ moves }}</div>
</template>

<script>
import { mapGetters } from 'vuex';

export default {
  name: 'Moves',
  computed: {
    ...mapGetters(['moves']),
  },
};
</script>
