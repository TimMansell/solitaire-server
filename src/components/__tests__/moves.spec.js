import { shallowMount } from '@vue/test-utils';
import Moves from '@/components/Moves.vue';

describe('Moves.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Moves, {
      computed: {
        moves: () => 1,
      },
    });

    expect(wrapper).toMatchSnapshot();
  });
});
