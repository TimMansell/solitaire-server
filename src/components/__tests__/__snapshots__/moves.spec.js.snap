// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Moves.vue matches snapshot 1`] = `
<div>
  Moves: <span data-test="moves">1</span></div>
`;
